### PR TITLE
fix: add certmanager and classname fields in ingress

### PR DESCRIPTION
**Pull request by pratapaprasanna**
fixes: [18207](https://github.com/appsmithorg/appsmith/issues/18207)

### CODE REVIEW

**Review by sharat87 — APPROVED** [review]

**Comment by sharat87** [comment]
Merging despite freeze, since this doesn't make any changes to application code.

**Comment by BhautikChudasama** [comment]
Upgradation fail due to `certManager` key not found.
```log
Helm upgrade failed: error validating "": error validating data: ValidationError(Ingress.spec): unknown field "certManager" in io.k8s.api.networking.v1.IngressSpec
```

File: deploy/helm/templates/ingress.yaml [hunk 1]
```diff
@@ -44,6 +44,8 @@ spec:
       secretName: {{ .secretName }}
     {{- end }}
   {{- end }}
+  certManager: {{ .Values.ingress.certManager }}
```
Comment (cep21) [inline deploy/helm/templates/ingress.yaml:47]: Failing on install: `unknown field "certManager"`

Is this a valid ingress?

**Comment by cep21** [comment]
I think this version is broken: has anyone gotten it to work?

**Pull request merged**